projects/steering/test/test_steering_can.c

```diff
@@ -0,0 +1,63 @@
+#include "can.h"
+#include "can_msg_defs.h"
+#include "event_queue.h"
+#include "exported_enums.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt_def.h"
+#include "ms_test_helpers.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "steering_can.h"
```

Comment:
Small nitpick: the coding standards specify that the #include for the module you're testing (in this case `steering_can.h`) should go at the very top of the includes, separated from the others by a blank line.